src/OSPSuite.Infrastructure.Import/Extensions/ColumnInfoExtensions.cs

```diff
@@ -13,5 +13,10 @@ public static bool IsAuxiliary(this ColumnInfo info)
       {
          return !info.IsMandatory;
       }
+
+      public static bool IsMeasurement(this ColumnInfo info)
```

Comment:
Why are those guys not defined in ColumnInfo (I mean all the extensions)?

Reply:
> Why are those guys not defined in ColumnInfo (I mean all the extensions)?

Furthermore, they should be getters. Moving logic already ;)